File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length==0) return 0;
        int left=0;
        for(int right=left+1;right<nums.length;right++){
            if(nums[left]==nums[right]) continue;
            else{
                nums[left+1] = nums[right];
                left++;
            }
        }
        return left+1;
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
#include<set>
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        std::set<int> numsFC;
        for(int i:nums){
            if(numsFC.contains(i)){
                return true;
            }
            numsFC.insert(i);
        }

        return false;
    }
};